lib/modules/manager/gradle/parser.ts

```diff
@@ -373,9 +373,11 @@ function processLibraryDep(input: SyntaxHandlerInput): SyntaxHandlerOutput {
 
   if (groupId && artifactId) {
     res.vars = { [key]: { key, value, fileReplacePosition, packageFile } };
-    const versionRefToken = tokenMap.version;
-    if (versionRefToken) {
-      const version: Token = { ...versionRefToken, type: TokenType.Word };
+    const version = tokenMap.version;
+    if (version) {
+      if (tokenMap.versionType?.value === 'versionRef') {
+        version.type = TokenType.Word;
+      }
```

Comment:
this will change the existing object instead of creating a new clone. can this cause new side effects?

@zharinov WDYT?

Reply:
Yes, would be great to clone too

Reply:
I didn't change this file. Some changes got added here while changing base from main -> renovate/4.x
Should I revert them?